core/capabilities/ccip/oraclecreator/plugin.go

```diff
@@ -131,7 +129,7 @@ func (i *pluginOracleCreator) Create(ctx context.Context, donID uint32, config c
 		return nil, fmt.Errorf("failed to get public config from OCR config: %w", err)
 	}
 
-	pluginConfig, err := initializerPluginConfig(destChainFamily, i.lggr)
+	pluginServices, err := initializerPluginServices(destChainFamily, i.lggr)
```

Comment:
No need for `initializerPluginServices`:
```suggestion
	pluginServices, err := ccipcommon.GetPluginServices(lggr, destChainFamily)
```